README.md

```diff
@@ -98,9 +92,11 @@ export DB_ENDPOINT=<database_endpoint>
 export DB_NAME=<database_name>
 ```
 
+Tip: if you're using GNU/Linux or MacOS: `printenv` to print the environment variables and check all configurations.
+
 5. Run the app:
 ```
-python run.py
+poetry run start
 ```
 
 6. Navigate to http://localhost:5000 in your browser
```

Comment:
a few lines below (i can't put a comment there), there are instructions for running on Linux, was that part supposed to go away? Since you removed the entry on the readme table of contents?

Reply:
@isabelcosta since the virtual env is being done by poetry the command would be same on both linux and windows I think. It needs to be confirmed though 👀